python/mxnet/image/image.py

```diff
@@ -79,6 +79,9 @@ def imread(filename, *args, **kwargs):
     >>> mx.img.imdecode(str_image, to_rgb=0)
     <NDArray 224x224x3 @cpu(0)>
     """
+    if not os.path.exists(filename):
```

Comment:
This should be done in c++

Reply:
How to do it in C++?

Reply:
yes. Looks like you already have a check in c++

Reply:
@piiswrong done